dicom-archive/dicomTar.pl

```diff
@@ -137,18 +137,19 @@
 my $creator         = $summary->{user};
 my $sumTypeVersion  = $summary->{sumTypeVersion}; 
 
-my $byDate;
 # Determine how to name the archive... by acquisition date or by today's date.
-if ($todayDate) { $byDate = $today; } else { $byDate = $summary->{header}->{scandate}; } # wrap up the archive 
-my $finalTarget = "$targetlocation/DCM_${byDate}_$summary->{metaname}.tar";
+my $byDate      = $todayDate ? $today : $summary->{header}->{scandate};
+my $pname       = quotemeta($summary->{header}->{pname});
+my $tarbasename = "${metaname}_$pname";
+my $finalTarget = "$targetlocation/DCM_${byDate}_$tarbasename.tar";
 
 if (-e $finalTarget && !$clobber) {
     print STDERR "\nTarget exists. Use clobber to overwrite!\n\n";
     exit $NeuroDB::ExitCodes::TARGET_EXISTS_NO_CLOBBER;
 }
 
 # read acquisition metadata into variable 
-my $metafile = "$targetlocation/$metaname.meta";
+my $metafile = "$targetlocation/$tarbasename.meta";
```

Comment:
@nicolasbrossard I investigated a little more about the meta file and the tmpdir and here is what I found:
- in dicomTar.pl, the metafile was always created in $targetlocation (see line above)
- line 130 of dicomTar.pl: `my $summary = DICOM::DCMSUM->new($dcm_source,$targetlocation);`
- new constructor of DCMSUM is: 
```
# The constructor 
sub new {
    my $proto = shift;
    my $class = ref($proto) || $proto;
    my $self  = {};
    bless ($self, $class);

    my $dcm_dir = shift;
    my $tmp_dir = shift;
```

=> this means that actually, $self->{tmpdir} = $targetlocation...

So, I could either leave things as I did or change the call to the function DCMSUM->database and replace $metafile by $tarbasename and revert the code in DCMSUM. Not sure what makes more sense.

Does that clarify the enigma? Let me know what you think.